platform/test/spec/testToolManagerSpec.js

```diff
@@ -44,7 +44,7 @@ describe("ToolManager", () => {
             // Check the expected results
             expect(tm.toolsUrls).toHaveSize(TOOL_URLS.length);
             
-            for (const i in TOOL_URLS){
+            for (let i = 0; i < TOOL_URLS.length; i++ ){
```

Comment:
Why is this change needed?

Reply:
The `for ... in` meant things other than array elements were being checked e.g. functions from the Array class using index only the elements are checked. I couldn't determine why after importing the playground there was difference in array objects created by the test compared to the one returned by the platform.

Reply:
Understood. This is one of the peculiarities of JS that I will never really get used to. Good to have this cleaned up.